<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-07-04T15:11:36 IST" name="Default suite" finished-at="2023-07-04T15:12:19 IST" duration-ms="42929">
    <groups>
    </groups>
    <test started-at="2023-07-04T15:11:36 IST" name="Default test" finished-at="2023-07-04T15:12:19 IST" duration-ms="42929">
      <class name="Runner_Script.TestRunner">
        <test-method is-config="true" signature="OpenAppln()[pri:0, instance:Runner_Script.TestRunner@1698fc68]" started-at="2023-07-04T15:11:36 IST" name="OpenAppln" finished-at="2023-07-04T15:11:53 IST" duration-ms="16643" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OpenAppln -->
        <test-method signature="test(java.lang.String,java.lang.String)[pri:0, instance:Runner_Script.TestRunner@1698fc68]" started-at="2023-07-04T15:11:53 IST" name="test" data-provider="testData" finished-at="2023-07-04T15:11:55 IST" duration-ms="2394" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Anju]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2345]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at Runner_Script.TestRunner.test(TestRunner.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method is-config="true" signature="CloseAppln(org.testng.ITestResult)[pri:0, instance:Runner_Script.TestRunner@1698fc68]" started-at="2023-07-04T15:11:55 IST" name="CloseAppln" finished-at="2023-07-04T15:12:03 IST" duration-ms="7308" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test status=FAILURE method=TestRunner.test(java.lang.String,java.lang.String)[pri:0, instance:Runner_Script.TestRunner@1698fc68] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseAppln -->
        <test-method is-config="true" signature="OpenAppln()[pri:0, instance:Runner_Script.TestRunner@1698fc68]" started-at="2023-07-04T15:12:03 IST" name="OpenAppln" finished-at="2023-07-04T15:12:12 IST" duration-ms="9586" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OpenAppln -->
        <test-method signature="test(java.lang.String,java.lang.String)[pri:0, instance:Runner_Script.TestRunner@1698fc68]" started-at="2023-07-04T15:12:12 IST" name="test" data-provider="testData" finished-at="2023-07-04T15:12:15 IST" duration-ms="2247" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Nandu]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[5678]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at Runner_Script.TestRunner.test(TestRunner.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method is-config="true" signature="CloseAppln(org.testng.ITestResult)[pri:0, instance:Runner_Script.TestRunner@1698fc68]" started-at="2023-07-04T15:12:15 IST" name="CloseAppln" finished-at="2023-07-04T15:12:19 IST" duration-ms="4693" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test status=FAILURE method=TestRunner.test(java.lang.String,java.lang.String)[pri:0, instance:Runner_Script.TestRunner@1698fc68] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseAppln -->
      </class> <!-- Runner_Script.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
